charts/gsp-cluster/templates/03-namespaces/namespace.yaml

```diff
@@ -13,6 +13,20 @@
     {{- end }}
   {{- end }}
 {{- end }}
+{{- $defaultedPermittedRolesRegex := .permittedRolesRegex | default "^$" }}
+{{- $permittedRolesRegex := printf "%s|^svcop-%s-%s-.*$" $defaultedPermittedRolesRegex $.Values.global.cluster.name .name }}
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
woah where did this come from?

Reply:
It looks like this was removed in #466 in favour of charts/gsp-cluster/templates/00-aws-auth/managed-namespaces.yaml added in #390

Reply:
thanks, fixed